lib/flows/delete-account-flow.js

```diff
@@ -3,21 +3,19 @@
 import CloseAccountPage from '../pages/account/close-account-page';
 import LoggedOutMasterbarComponent from '../components/logged-out-masterbar-component';
 import * as SlackNotifier from '../slack-notifier';
-import NavBarComponent from '../components/nav-bar-component';
 import AccountSettingsPage from '../pages/account/account-settings-page';
-import ProfilePage from '../pages/profile-page';
 
 export default class DeleteAccountFlow {
 	constructor( driver ) {
 		this.driver = driver;
 	}
 	async deleteAccount( name ) {
 		return ( async () => {
-			const navBarComponent = await NavBarComponent.Expect( this.driver );
-			await navBarComponent.clickProfileLink();
-			const profilePage = await ProfilePage.Expect( this.driver );
-			await profilePage.chooseAccountSettings();
-			const accountSettingsPage = await AccountSettingsPage.Expect( this.driver );
+			let url = await this.driver.getCurrentUrl();
+			if ( url.startsWith( 'data:' ) === true ) {
+				await this.driver.sleep( 2000 );
```

Comment:
Wait 2s if test brakes and after that URL is `data:`. I didn't find another way to wait and open AccountSettingsPage().